src/components/MenuItem.js

```diff
@@ -61,20 +76,35 @@ const MenuItem = ({
             <>
                 <View style={styles.flexRow}>
                     {icon && (
-                        <View style={styles.createMenuIcon}>
-                            <Icon src={icon} fill={getIconFillColor(getButtonState(hovered, pressed, success))} />
-                        </View>
+                    <View style={styles.createMenuIcon}>
+                        <Icon
+                            src={icon}
+                            width={iconWidth}
+                            height={iconHeight}
+                            fill={getIconFillColor(getButtonState(hovered, pressed, success))}
+                        />
+                    </View>
                     )}
                     <View style={[styles.justifyContentCenter, styles.menuItemTextContainer]}>
                         <Text style={[styles.createMenuText, styles.ml3]}>
                             {title}
                         </Text>
+                        {description && (
+                            <Text style={[styles.createMenuDescription, styles.ml3, styles.mt1]}>
+                                {description}
+                            </Text>
+                        )}
                     </View>
                 </View>
                 {shouldShowRightIcon && (
-                    <View style={styles.createMenuIcon}>
-                        <Icon src={iconRight} fill={getIconFillColor(getButtonState(hovered, pressed))} />
-                    </View>
+                <View style={styles.createMenuIcon}>
+                    <Icon
+                        src={iconRight}
+                        width={iconWidth}
+                        height={iconHeight}
```

Comment:
To clarify, the `iconRight` should also take the sizing information for the left icon? But we are not using it in the proposed design? If correct, I think we don't need to make this change today.

Reply:
Removed!